### DISCUSSION TITLE
Can we support Tensorflow 1.15?

### DISCUSSION THREAD

**Discussion by wangty6** [arogozhnikov/einops#244] [General]
Currently it will give [module 'tensorflow.python.keras.api._v1.keras.backend' has no attribute 'is_keras_tensor'] error, and I have to replace is_keras_tensor to False.

**Comment by arogozhnikov** [comment]
TF < 2 was supported in earlier versions (0.2, maybe 0.3)

**Reply by wangty6** [reply]
Thanks for you reply, I wonder if there's a way we can let TF<2 benefit from latest update?

**Reply by arogozhnikov** [reply]
TF < 2 support is over. If you want to use newer features, you'll need to upgrade to TF2.

**Reply by wangty6** [reply]
Our distributed training infra strongly bind to TF 1.15, I think it cannot upgrade in a short period. Anyway, thank you for your quick reply. We will use earlier versions of einops or manually hack currently version.
